services/management/checks_test.go

```diff
@@ -109,3 +109,41 @@ func TestGetSecurityCheckResults(t *testing.T) {
 		assert.Equal(t, resp, response)
 	})
 }
+
+func TestListSecurityChecks(t *testing.T) {
+	t.Run("get disabled checks error", func(t *testing.T) {
```

Comment:
why is there no test case for the success condition?

Reply:
lets add one otherwise around 10 lines of code are going uncovered for no reason

Reply:
Agree, done.